connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -125,7 +125,7 @@ class SparkConnectService(debug: Boolean)
       SparkConnectService
         .getOrCreateIsolatedSession(userId, sessionId)
         .session
-    val stackTraceEnabled = session.conf.get(PYSPARK_JVM_STACKTRACE_ENABLED.key, "true").toBoolean
+    val stackTraceEnabled = session.conf.get(PYSPARK_JVM_STACKTRACE_ENABLED)
```

Comment:
`Spark Shell` also shows JVM StackTrace by default, doesn't it, @ueshin ?
> Users won't see the JVM stack trace by default.

Reply:
I'm wondering if this PR aims to match with `PySpark Shell` environment or not.

Reply:
That's a good point.

The original PR is for Spark Connect Python client and it was supposed to be disabled by default if it's not testing mode:
https://github.com/apache/spark/blob/939c5f507b1d3225d9575ae39ab494df092c6c9e/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L2780-L2789

For Scala client, I think it's TBD? https://github.com/apache/spark/pull/40575#issuecomment-1512439357 cc @allisonwang-db

Reply:
Thank you!

Reply:
Yea this `PYSPARK_JVM_STACKTRACE_ENABLED` flag is Python-specific.